### DISCUSSION TITLE
multiplayer not loading help pls

### DISCUSSION THREAD

**Discussion by afk4747** [ppy/osu#16444] [Q&A]
how do i fix my multiplayer cuz its not loading pls help

**Comment by bdach** [comment]
please read https://github.com/ppy/osu/discussions/15188 first

i don't know what "its not loading" means

**Reply by afk4747** [reply]
like it dose not show any rooms in the multiplayer search

**Reply by bdach** [reply]
i will take that as a no on "please read (above link) first"

**Reply by frenzibyte** [reply]
I'm not even sure if it's "not loading" or there are basically no rooms created or you have the wrong ruleset selected.

**Reply by Moltanicaa** [reply]
One player wrote to me and gave me this screen, so yes, there is a problem. I'll try to ask for his logs (everything is fine with me).

The problem is the infinite loading of the multiplayer.
![yiTmovBZojM](https://user-images.githubusercontent.com/59089206/152654702-f42ec9e8-dc0a-4c12-8588-858a7379249c.jpg)

UPD. Logs:
[runtime.log](https://github.com/ppy/osu/files/8008800/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8008801/updater.log)
[performance.log](https://github.com/ppy/osu/files/8008802/performance.log)
[network.log](https://github.com/ppy/osu/files/8008803/network.log)
[database.log](https://github.com/ppy/osu/files/8008804/database.log)

**Reply by peppy** [reply]
The provided logs don't even enter the multiplayer screen.
